cea/demand/thermal_loads.py

```diff
@@ -81,37 +79,33 @@ def calc_thermal_loads(building_name, bpr, weather_data, usage_schedules, date,
         ventilation_air_flows_simple.calc_m_ve_leakage_simple(bpr, tsd, gv)
 
         # get internal comfort properties
-        tsd = controllers.calc_simple_temp_control(tsd, bpr.comfort, gv.seasonhours[0] + 1, gv.seasonhours[1],
-                                                   date.dayofweek)
-
-
+        tsd = control_heating_cooling_systems.calc_simple_temp_control(tsd, bpr, date.dayofweek)
 
         # end-use demand calculation
-        for t in range(-720, 8760):
-            hoy = helpers.seasonhour_2_hoy(t, gv)
+        for t in get_hours(bpr):
 
             # heat flows in [W]
             # sensible heat gains
-            tsd = sensible_loads.calc_Qgain_sen(hoy, tsd, bpr, gv)
+            tsd = sensible_loads.calc_Qgain_sen(t, tsd, bpr, gv)
 
             if use_dynamic_infiltration_calculation:
                 # OVERWRITE STATIC INFILTRATION WITH DYNAMIC INFILTRATION RATE
                 dict_props_nat_vent = ventilation_air_flows_detailed.get_properties_natural_ventilation(bpr, gv)
                 qm_sum_in, qm_sum_out = ventilation_air_flows_detailed.calc_air_flows(
-                    tsd['T_int'][hoy - 1] if not np.isnan(tsd['T_int'][hoy - 1]) else tsd['T_ext'][hoy - 1],
-                    tsd['u_wind'][hoy], tsd['T_ext'][hoy], dict_props_nat_vent)
+                    tsd['T_int'][t - 1] if not np.isnan(tsd['T_int'][t - 1]) else tsd['T_ext'][t - 1],
+                    tsd['u_wind'][t], tsd['T_ext'][t], dict_props_nat_vent)
                 # INFILTRATION IS FORCED NOT TO REACH ZERO IN ORDER TO AVOID THE RC MODEL TO FAIL
-                tsd['m_ve_inf'][hoy] = max(qm_sum_in / 3600, 1 / 3600)
+                tsd['m_ve_inf'][t] = max(qm_sum_in / 3600, 1 / 3600)
```

Comment:
this seems dangerous... you probably have `from __future__ import division` so this works, but double-check that.
also, I guess you could just set it to `max(qm_sum_in, 1) / 3600` and then add a constant for 3600 - is this a conversion?

Reply:
Yes. it is a conversion.
didn't know that this can be done.

Reply:
double-checked: you have `from __future__ import division`, so, we leave this :-)